crates/ruff_linter/src/rules/pydoclint/rules/check_docstring.rs

```diff
@@ -210,7 +210,7 @@ fn parse_entries_numpy(content: &str) -> Vec<QualifiedName> {
     for potential in split {
```

Comment:
While we're here, can we use `content.lines()` above instead of `content.split`? I'd hope `content.lines()` will also handle `\r`.